internal/daysteps/daysteps.go

```diff
@@ -1,23 +1,66 @@
 package daysteps
 
 import (
+	"errors"
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
 	"time"
+
+	"github.com/Yandex-Practicum/go1fl-4-sprint-final/internal/spentcalories"
 )
 
 var (
-	StepLength = 0.65 // длина шага в метрах
+	StepLength = 0.65
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// ваш код ниже
+	slice := strings.Split(data, ",")
+	if len(slice) != 2 {
+
+		return 0, 0, fmt.Errorf("the number of parameters must be 2")
+	}
+
+	steps, err := strconv.Atoi(slice[0])
+	if err != nil {
+
+		return 0, 0, fmt.Errorf("сan not convert string into integer")
```

Comment:
Теряешь сообщение об ошибки
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```

Reply:
Нужно везде это поправить